specification/agrifood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json

```diff
@@ -1121,6 +1128,13 @@
       "description": "FarmBeats update request.",
       "type": "object",
       "properties": {
+        "sku": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Sku",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
```

Comment:
It looks like this schema is only used as the request body for patch, which is documented as "Update" for the FarmBeats resource. Can this patch operation also create a FarmBeats resource? If not, then there is no need -- and more likely confusing -- to add "sku" to the request model here with "x-ms-mutability: [read, create]".

Said another way -- this schema is only used for update, so a property only allowed for create has no purpose here.

Reply:
removed as suggested